ingestion/src/metadata/ingestion/source/metadata/atlas.py

```diff
@@ -160,41 +164,55 @@ def _parse_table_entity(self, name, entity):
                     db_entity = tbl_entity["relationshipAttributes"][
                         self.service_connection.entityTypes["Table"][name]["db"]
                     ]
-                    db = self.get_database_entity(db_entity["displayText"])
+                    database_request = self.get_database_entity(
+                        db_entity["displayText"]
+                    )
                     table_name = tbl_attrs["name"]
                     tbl_description = tbl_attrs["description"]
+                    yield database_request
 
-                    tbl_attrs = tbl_entity["attributes"]
-                    db_entity = tbl_entity["relationshipAttributes"]["db"]
-                    fqn_obj = fqn.build(
+                    database_fqn = fqn.build(
                         self.metadata,
-                        entity_type=Table,
-                        service_name=self.config.serviceName,
-                        database_name=db.name.__root__,
-                        schema_name=db_entity["displayText"],
-                        table_name=table_name,
+                        entity_type=Database,
+                        service_name=self.config.serviceConnection.__root__.config.dbService,
+                        database_name=database_request.name.__root__,
                     )
-                    om_table_entity = Table(
-                        id=uuid.uuid4(),
-                        name=table_name,
-                        description=tbl_description,
-                        fullyQualifiedName=fqn_obj,
-                        columns=tbl_columns,
+
+                    database_object = self.metadata.get_by_name(
+                        entity=Database, fqn=database_fqn
                     )
-                    database_schema = DatabaseSchema(
+
+                    tbl_attrs = tbl_entity["attributes"]
+                    db_entity = tbl_entity["relationshipAttributes"]["db"]
+
+                    database_schema_request = CreateDatabaseSchemaRequest(
                         name=db_entity["displayText"],
-                        service=EntityReference(
-                            id=self.service.id, type=self.service_connection.serviceType
+                        database=EntityReference(
+                            id=database_object.id, type="database"
                         ),
-                        database=EntityReference(id=db.id.__root__, type="database"),
                     )
+                    yield database_schema_request
 
-                    table_and_db = OMetaDatabaseAndTable(
-                        table=om_table_entity,
-                        database=db,
-                        database_schema=database_schema,
+                    database_schema_fqn = fqn.build(
+                        self.metadata,
+                        entity_type=DatabaseSchema,
+                        service_name=self.config.serviceConnection.__root__.config.dbService,
```

Comment:
you can use `self.service_connection` instead of `self.config.serviceConnection.__root__.config` here

Reply:
Ok, will done.